synapse/lib/layer.py

```diff
@@ -282,10 +282,10 @@ def __init__(self, layr, form):
         self.form = form
 
     def getNodeValu(self, buid):
-        bkey = buid + b'\x00'
-        byts = self.layr.layrslab.get(bkey, db=self.layr.bybuid)
-        if byts is not None:
-            return s_msgpack.un(byts)[1]
+        sode = self.layr._getStorNode(buid)
+        valt = sode.get('valu')
```

Comment:
valt vs valu? just a odd style nit but it seems weird to have a inconsistent label in the codebase